### ISSUE TITLE
[REQUEST] Apnonce setter integrated into Checkra1n

### ISSUE THREAD

**Issue by xrotorhead** [checkra1n/BugTracker#347]
This would be great for those trying to downgrade from 13.2.2 to 13.1.3.

**Comment by quilenle** [comment]
I agree if we integrate the APNonce setter.
I have researched on the internet for the APNonce setter that based on checkra1n/checkm8 but it seems like it didn't support all devices. It will be great if we can integrate the APNonce setter in checkra1n, so we wouldn't worry about boot loop since we can use futurerestore with the default nonce is 0x1111111111111111. Cause we can't set the nonce through the command line like before. Furthermore, my idea for this is we have an input text for the nonce on the UI, with the default nonce is 0x1111111111111111 (we need to validate if the nonce that inputted was 0x[something] or not).
In conclusion, this will be grateful if we can implement the APNonce setter on the checkra1n UI.
Best regards.

**Comment by quilenle** [comment]
Hi @sbingner , could you please check this request.
Sorry if I was annoyed you.

**Comment by Siguza** [comment]
Duplicate of #181.
